update/Pos/Update/Configuration.hs

```diff
@@ -70,5 +85,24 @@ lastKnownBlockVersion = ccLastKnownBlockVersion updateConfiguration
 curSoftwareVersion :: HasUpdateConfiguration => SoftwareVersion
 curSoftwareVersion = SoftwareVersion ourAppName (ccApplicationVersion updateConfiguration)
 
+-- | @SystemTag@ corresponding to the operating system/architecture pair the program was
+-- compiled in.
+-- The @Distribution.System@ module from @Cabal@ was used
+-- (https://hackage.haskell.org/package/Cabal-2.0.1.1/docs/Distribution-System.html)
+currentSystemTag :: SystemTag
+currentSystemTag =
+    $(do let s :: Either String SystemTag
+             s = mkSystemTag (toText (osHelper buildOS ++ archHelper buildArch))
+             color col t =
```

Comment:
Please use functions from `Serokell.Util.ANSI` to colorize text.

Reply:
@gromakovsky the last time I worked with this was in `log-warper` so I was unsure there were functions in `serokell-util`. I'll do that.